packages/faceted-search/src/components/Badges/BadgeOperator/BadgeOperator.scss

```diff
@@ -23,8 +23,19 @@ $circle-size-small: 1.8rem !default;
 		> button {
 			text-decoration: none;
 			text-transform: none;
+			padding: 0;
 		}
 	}
+
+	/* only effect in IE11*/
+	@media screen and(-ms-high-contrast:active), (-ms-high-contrast: none) {
```

Comment:
Had to ask google to understand why, kind of a tricky way to target only ie10+, but don't found anything else that looks better so 👍